examtool/examtool/cli/compile.py

```diff
@@ -71,6 +71,11 @@
     default=False,
     help="Generates a draft copy of the exam, which is faster but less accurate.",
 )
+@click.option(
```

Comment:
Help is confusing, because it doesn't correspond to the True value of the flag.

Reply:
Changed to a different flag name.